docs/getting-started/create-defend-policy-api.asciidoc

```diff
@@ -0,0 +1,813 @@
+                },
+                "behavior_protection": {
+                  "mode": "prevent",
+                  "supported": true
+                },
+                "memory_protection": {
+                  "mode": "prevent",
+                  "supported": true
+                },
+                "popup": {
+                  "malware": {
+                    "message": "",
+                    "enabled": true
+                  },
+                  "behavior_protection": {
+                    "message": "",
+                    "enabled": true
+                  },
+                  "memory_protection": {
+                    "message": "",
+                    "enabled": true
+                  }
+                },
+                "logging": {
+                  "file": "info"
+                }
+              },
+              "linux": {
+                "events": {
+                  "process": true,
+                  "file": true,
+                  "network": true,
+                  "session_data": false,
+                  "tty_io": false
+                },
+                "malware": {
+                  "mode": "prevent",
+                  "blocklist": true
+                },
+                "behavior_protection": {
+                  "mode": "prevent",
+                  "supported": true
+                },
+                "memory_protection": {
+                  "mode": "prevent",
+                  "supported": true
+                },
+                "popup": {
+                  "malware": {
+                    "message": "",
+                    "enabled": true
+                  },
+                  "behavior_protection": {
+                    "message": "",
+                    "enabled": true
+                  },
+                  "memory_protection": {
+                    "message": "",
+                    "enabled": true
+                  }
+                },
+                "logging": {
+                  "file": "info"
+                }
+              }
+            }
+          }
+        }
+      }
+    ],
+    "revision": 1,
+    "created_at": "2023-04-06T15:53:14.020Z",
+    "created_by": "elastic",
+    "updated_at": "2023-04-06T15:53:14.020Z",
+    "updated_by": "elastic"
+  }
+}
+----
+<1> `<PACKAGE-POLICY-ID>` needed in step 3
+
+[discrete]
+[[customize-policy-settings]]
+== Step 3: Customize the {elastic-defend} policy settings
+
+With the response received in step 2, do the following:
+
+. Pull out all of the content under the top level `item` object.
+. Remove the following fields:
++
+[source, json]
+----
+"id": "<PACKAGE-POLICY-ID>",
+"revision": 1,
+"created_at": "2023-04-06T15:53:14.020Z",
+"created_by": "elastic",
+"updated_at": "2023-04-06T15:53:14.020Z",
+"updated_by": "elastic"
+----
+. Make any changes to the `policy` object to customize the {elastic-defend} configuration.
+
+Using the resulting data, save the modified {elastic-defend} policy.
```

Comment:
```suggestion
Include the resulting JSON object in the following call to modify the {elastic-defend} policy. Replace these fields:
```
Maybe my suggestion is too wordy, but I found this sentence pretty confusing in isolation because it didn't specify how to do the thing it was telling me how to do. I had to skip several lines down to figure out how to do what it was telling me

Reply:
I can see how @benironside's edit helps clarify things. In addition, this line feels like it's another step in the numbered procedure above, but it's formatted as plain text. And then there's another stepped procedure below this -- maybe combine all into a continuous stepped procedure?

Reply:
Agree this could have been more clear. I've used subsections to rearrange the content, which should hopefully make it clearer that step 3 consists of two sub-steps (1. modify the default policy configuration and 2. save the customized configuration).